journal/modernized/09-chapter-7.adoc

```diff
@@ -88,19 +88,19 @@ which man was in before he transgressed the royal law of God.
 But now he, who kept not his first habitation, neither abode in the truth,
 envied man`'s happiness in the truth, and therefore, as a serpent,
 more subtle than any beast of the field, which the Lord God had made, came to the woman,
-who out of the man was made, and said, Yes, has God said,
-you shall not eat of every tree of the garden?
-The woman said, We may eat of the fruit of the trees of the garden;
+who out of the man was made, and said, "`Yes, has God said,
+you shall not eat of every tree of the garden?`"
+The woman said, "`We may eat of the fruit of the trees of the garden;
 but of the fruit of the tree which is in the midst of the garden, God has said,
-you shall not eat of it, neither shall you touch it, lest you die.
-And the serpent said unto the woman, you shall not surely die; for God does know,
+you shall not eat of it, neither shall you touch it, lest you die.`"
+And the serpent said unto the woman, "`You shall not surely die; for God does know,
 that in the day you eat thereof, then your eyes shall be opened,
-and you shall be as gods, knowing good and evil.
+and you shall be as gods, knowing good and evil.`"
 And here the serpent got an entrance.
-And when the woman, saw that the tree was good for food,
+And when the woman, (mark) saw that the tree was good for food,
```

Comment:
I just happened to see these as I was adding quotation marks so I added them back in from the original, idk if you wanted that but I know GF has a lot of them in his writing too so I thought they might be important. Let me know if you want me to take them out again.